teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| Global - Currently Active Duty | 42 | Hit | User responds yes or no on the edu-0994-submission-successful event label on either the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page or the www.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/review-and-submit/ page |
+| Global - Will Be Called to Active Duty | 43 | Hit | User responds no on the edu-0994-sip-form on either the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page or www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/confirmation/ page |
+| Education - Education Completed | 44 | Hit | How a user responds to the level of education completed on the edu-0994 form. Option include: (1) some college (2) bachelors degree (3) masters degree (4) associates degree (5) high school diploma or GED (6) other (7) doctoral degree (8) some high school  |
+| Global - Salary | 45 | Hit | How a user responds to the salary range question on the edu-0994 form. Option include: (1) more than 75k (2) 50k to 75k (3) 35k to 50k (4) 20k to 35k (5) less than 20k |
+| Education 22-0994 Applied Past Benefits | 46 | Hit | User response to the applied past benefits question (yes or no) collected on the edu-0994-submission-successful event label on the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page |
+| Education 22-0994 Currently Working Industry | 47 | Hit | User response to the currently working industry question (yes or no) collected on the edu-0994-submission-successful event label on the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page |
+| Education 22-0994 Made Program Selection | 48 | Hit | User response to the made program question (yes or no) collected on the edu-0994-submission-successful event label on the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page |
+| Education - STEM Scholarship Applicant | 49 | Hit | User response to STEM scholarship question (yes or unanswered) collected with the edu-howToApply-applyNow event label |
+| Profile Section Saved - Direct Deposit | 50 | User | User who is logged in saved their direct deposit information |
+| Health Facility Location | 51 | Hit | Name of hte health facility location that the event/pageview/action is taking place (for example, Pittsburgh on this page: www.va.gov/pittsburgh-health-care/) |
+| Health Facility Type | 52 | Hit | This CD is set as `Campus` when a user visits or completes actions on this page:         <br>www.va.gov/careers-employment/vetsuccess-on-campus/  |
```

Comment:
Perhaps we consider splitting out a 4th "example" column and keep this column as strictly a definition....your specific pages are great and we should utilize those as examples